packages/graphql/src/mutations/marketplace/createListing.js

```diff
@@ -4,13 +4,12 @@ import validator from '@origin/validator'
 import txHelper, { checkMetaMask } from '../_txHelper'
 import contracts from '../../contracts'
 import cost from '../_gasCost'
-import dapp1Compatibility from './_dapp1Compat'
 
 export function listingInputToIPFS(data, unitData, fractionalData) {
   const listingType = fractionalData ? 'fractional' : 'unit'
   const ipfsData = {
     __typename: data.typename || 'UnitListing',
-    schemaId: 'https://schema.originprotocol.com/listing_1.0.0.json',
+    schemaId: 'https://schema.originprotocol.com/listing_2.0.0.json',
```

Comment:
Why is it necessary to change version from 1.0.0 to 2.0.0 ?
Is that going to create issues for reading/writing legacy listings created using 1.0.0 ?

Ahh.. is that @wanderingstan changes for hourly listings ?

Reply:
Yes, that was @nick and I. These are breaking changes to the schema for fractional listings. However, old listings should be automatically converted by the event sourcer, but we need to do a lot of testing on that.